libs/k8s-client/provisioning/local/kubernetes/main.tf

```diff
@@ -46,21 +46,16 @@ resource "kubernetes_role" "k8s_client" {
     name      = "k8s-client"
   }
 
-  rule {
-    api_groups = [""]
-    resources  = ["events"]
-    verbs      = ["get", "list"]
-  }
-
   rule {
     api_groups = [""]
     resources  = [
       "configmaps",
+      "events",
       "persistentvolumeclaims",
       "pods",
       "secrets",
     ]
-    verbs      = ["*"]
+    verbs      = ["get", "list", "delete", "create", "patch", "deletecollection"]
```

Comment:
Myslím že ty oprávnění byly moc benevolentní. Takhle by to mělo mít vše co potřebuje.

Reply:
`*` jsou komplet oprávnění ne? `+ watch, attach, log, ....`

Reply:
Jo to jo, myslel jsem to spis z pohledu security - kdyz uz umis get/list, tak ze nemuzes watch ti uz nic nezachrani. A treba `logs` jsou specificky jen pro `pods` (nic jinyho AFAIK logy nema) a budes je muset nejspis zase pridat, jakmile se udela test na `PodsApiClient::readLog()`

Reply:
Mas ale pravdu, ze treba `attach` je zradny.. to jsem si neuvedomil, ze je jako separatni permission

Reply:
Podle mě by to mělo mít nejmenší oprávnění co potřebuje. Až budeme potřebovat logs, tak se prostě přidá. Děláme to tak i jinde že nepovolujeme všechno.